blueprints/k0s-lens-appiq/k0s-lens-appiq.yaml

```diff
@@ -34,8 +34,8 @@ spec:
           version: 0.1.0
           values: |
             auth:
-              adminUser: "$LAIQ_USER" # Required, must be set
-              adminPassword: "$LAIQ_PASSWORD" # Required, must be set
+              adminUser: "admin" # Required. This should be changed
```

Comment:
any reason we are not using env variable vs hardcoded credentials? From example perspective, we should provide env variable to showcase how these should indeed be run.

Reply:
Only to simplify the example and avoid questions from users that it doesn't work because they didn't set the env vars or used an invalid password. Especially since the error message is not so obvious.

An option might be to get a section added to https://mirantiscontainers.github.io/boundless/docs/blueprint-reference/ that talks about using env vars in a blueprint and mention it in this example as best practices.

Another option could be to get https://mirantis.jira.com/browse/BOP-76 done and as part of it throw an error or warning when an env var wasn't found that is being used in a blueprint.

Reply:
@nwneisen Good points. Ok, lets keep it simple enough but add somewhere in docs/example so users know how to pass env variable. Maybe put it in the same example as comment.

Then, +1 to both your suggestion. Throwing an error if an env variable doesn't resolve is key to user experience, the cost of not doing is, passing in empty values that might result in to components failing later on or some unpredictable component runtime behavior.

Reply:
Sounds good. I'm still looking at an issue with a PVC getting created in kind but not in k0s and I haven't had time to determine why. I'll make sure and add the extra comments to this before it gets merged in.